"IMAGE","FILE","TASK",ERROR"
"anchore/test_images:bin","/staged/copy/busybox","unknowns-labeler","no package identified in executable file"
"anchore/test_images:bin","/staged/copy/go","unknowns-labeler","no package identified in executable file"
"anchore/test_images:bin","/staged/copy/libpython2.7.so.1.0","unknowns-labeler","no package identified in executable file"
"anchore/test_images:bin","/staged/copy/python3.8","unknowns-labeler","no package identified in executable file"
"anchore/test_images:bin","/usr/lib/go/src/runtime/race/race_darwin_amd64.syso","cargo-auditable-binary-cataloger","rust cataloger: unable to read dependency information: unknown file format"
